expose.go

```diff
@@ -0,0 +1,130 @@
+package chromedp
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/chromedp/cdproto/page"
+	"github.com/chromedp/cdproto/runtime"
+)
+
+// ExposedFunc is the function type that can be exposed to the browser env.
+type ExposedFunc func(args string) (string, error)
```

Comment:
I think it would be more flexible if the args parameter was defined as an interface{} type.

Reply:
The reason for this design is that it is consistent with the cdp bindingCalled event parameter, which can only pass string.
Refer to ``Runtime.bindingCalled`` 
https://chromedevtools.github.io/devtools-protocol/tot/Runtime/#event-bindingCalled
```
PARAMETERS
    name string
    payload string

var payload struct {
    Type string `json:"type"`
    Name string `json:"name"`
    Seq  int64  `json:"seq"`
    Args string `json:"args"`
}```